tests/test_views.py

```diff
@@ -36,21 +37,25 @@ def mock_redirect(mocker):
 
 
 @pytest.mark.django_db
-def test_client_or_error_no_config(mock_request, mock_create_client):
+def test_client_or_error_no_config(mocker, mock_request, mock_create_client):
+    spy = mocker.spy(DefaultHooks, "system_error")
     response = _client_or_error(mock_request)
 
     mock_create_client.assert_not_called()
+    spy.assert_called_once_with(ANY, ANY, "init")
```

Comment:
This is a really cool pattern!

Reply:
Would it makes sense to import the `Operation` class though and not have more instances of these strings hanging around?

Reply:
Yep, great idea. Done in https://github.com/Office-of-Digital-Services/django-cdt-identity/pull/49/commits/52dcb0ca5ef24f2ac450beeb5a21614eb8163a6f